### ISSUE TITLE
Editable cell in disabled column

### ISSUE THREAD

**Issue by rohitiiit** [joffreyBerrier/vue-spreadsheet#193]
Hi,

Where the whole column is made in-editable (disabled), can we make few cells in that column editable?
Also is the comment section editable?

**Comment by joffreyBerrier** [comment]
Hi,

Could you rephrase your question. I don't understand, do you have an example ?

What is "whole column" ? do you mean the entire row ?

thx

**Comment by rohitiiit** [comment]
There is a disabled option which takes array of columns which will make
those columns disabled or non editable.
Now for all rows, the cells under those columns will remain disabled/non
editable.
Please correct me if I am wrong here.

Now i want the control on cell level. Suppose i have 2 columns A and B.
Now for first row i want cell B (or cell in column B) as disabled, but for
2nd row i want cell in column B as editable.

Am I clear now? Is it possible?

**Comment by joffreyBerrier** [comment]
Yes it's clear @rohitiiit ,

The `disableCells` property permit to disabled all the cell on a specific header (a, b ...) if you want to disabled on specific cell you have to use the attribute `disabled` directly on the cell.

i used your example below

```
data: {
  headers: [
    {
      headerName: 'A',
      headerKey: 'a',
      style: {
        width: '200px',
        minWidth: '200px',
      },
    },
    {
      headerName: 'B',
      headerKey: 'b',
      style: {
        width: '200px',
        minWidth: '200px',
      },
    },
  ],
  products: [
    {
      a: {
        type: 'input',
        value: 'first row (A) enable',
        active: false,
        disabled: false,
      },
      b: {
        type: 'input',
        value: 'first row (B) disable',
        active: false,
        disabled: true,
      },
    },
    {
      a: {
        type: 'input',
        value: 'second row (A) enable',
        active: false,
        disabled: false,
      },
      b: {
        type: 'input',
        value: 'second row (B) enable',
        active: false,
        disabled: false,
      },
    },
  ],
}
```

i hope i'm clear thx

**Comment by rohitiiit** [comment]
Thanks for prompt answer. Will try that.

Raising issue in GitHub is the right way to ask question like I did or
should I use some other way? I might have few more questions later.

**Comment by joffreyBerrier** [comment]
It is probably not the goodway but today i don't have another way to respond, so keep going like this :) !

Add the **question** label to the issue like i did

thx
